image_dehazer/__init__.py

```diff
@@ -32,12 +32,12 @@ def __AirlightEstimation(self, HazeImg):
     def __BoundCon(self, HazeImg):
         if (len(HazeImg.shape) == 3):
 
-            t_b = np.maximum((self._A[0] - HazeImg[:, :, 0].astype(float)) / (self._A[0] - self.C0),
-                             (HazeImg[:, :, 0].astype(float) - self._A[0]) / (self.C1 - self._A[0]))
-            t_g = np.maximum((self._A[1] - HazeImg[:, :, 1].astype(float)) / (self._A[1] - self.C0),
-                             (HazeImg[:, :, 1].astype(float) - self._A[1]) / (self.C1 - self._A[1]))
-            t_r = np.maximum((self._A[2] - HazeImg[:, :, 2].astype(float)) / (self._A[2] - self.C0),
-                             (HazeImg[:, :, 2].astype(float) - self._A[2]) / (self.C1 - self._A[2]))
+            t_b = np.maximum((self._A[0] - HazeImg[:, :, 0].astype(float)) / (self._A[0] - self.C0 + 1e-6),
```

Comment:
Can you use `eps_float64 = np.finfo(float).eps` instead of hardcoding the 1e-6?

Reply:
Of Course. `np.finfo(float).eps` is more accurate than hardcoding a value like 1e-6.